### PR TITLE
kick off

### PR DESCRIPTION
**Pull request by Casanova569**
Summary:
This is the first steps in the process to create a netcore proyect.
Objetive: The trello card is considered approved if the default environment can be lifted on the localhost.

Trello card:
Name: Kick-Off
Link: https://trello.com/c/7BCYh54j/1-kick-off

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (truccomarcos) [inline .vscode/launch.json:1]: Don't forget to add this folder to `.gitignore` (look up how to untrack specific files that have been added to the repository)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bin
+obj
```
Comment (truccomarcos) [inline .gitignore:2]: Add an empty line at the end of the file (git doesn't like it when files don't end on a blank line :P )
image

**Review by truccomarcos — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 1 above)
Comment (Casanova569) [reply .vscode/launch.json:1]: I fixed the error in the next branch - MVC (controller).

File: .gitignore (hunk 2 above)
Comment (Casanova569) [reply .gitignore:2]: I fixed the error in the next branch - MVC (controller).

**Review by truccomarcos — APPROVED** [review]

**Review by mikolaslorant — APPROVED** [review]

**Pull request merged**